Cargo.toml

```diff
@@ -470,6 +478,10 @@ alloy-transport-http = { version = "0.6.4", features = [
 alloy-transport-ipc = { version = "0.6.4", default-features = false }
 alloy-transport-ws = { version = "0.6.4", default-features = false }
 
+# scroll
+# TODO (scroll): point to crates.io/tag once the crate is published/ a tag is created.
+poseidon-bn254 = { git = "https://github.com/scroll-tech/poseidon-bn254", branch = "master", features = ["bn254"] }
```

Comment:
Should we use a commit for now (instead of a branch)? So that things don't break unexpectedly.

Reply:
that's fine with me, I was pointing it to `master` because this is what is currently done on [`scroll-tech/revm`](https://github.com/scroll-tech/revm/blob/scroll-evm-executor/v49/crates/primitives/Cargo.toml#L60). I can check what commit is used in `scroll-tech/revm` in the lock file and use that one for now.